.pylintrc

```diff
@@ -6,6 +6,7 @@ disable=too-few-public-methods
 [MESSAGES CONTROL]
 # raising 'from' is almost entirely false positives
 disable=raise-missing-from
+#fail-on=unused-variable,comparison-with-itself,self-assigning-variable
```

Comment:
This looks commented out?

Reply:
It was, because I was testing it and forgot to uncomment it. It's fixed now.